core/tests/data/third_party/requirements_test.txt

```diff
@@ -0,0 +1,14 @@
+#
+# This file is a test requirements.txt file that is used to test the
+# functionality of install_backend_python_libs._get_requirements_file_contents()
+
+dependency1==1.6.1      # via -r requirements.in, dependency3
+dependency2==4.9.1      # via -r requirements.in
+dependency3==3.1.5      # via -r requirements.in
```

Comment:
If it matters, maybe test something with a different version schema to change things up.